src/Data/Array/NonEmpty.purs

```diff
@@ -350,15 +353,49 @@ span
   -> { init :: Array a, rest :: Array a }
 span f = adaptAny $ A.span f
 
+-- | Group equal, consecutive elements of an array into arrays.
+-- |
+-- | ```purescript
+-- | group (NonEmptyArray [1, 1, 2, 2, 1]) ==
+-- |   NonEmptyArray [NonEmptyArray [1, 1], NonEmptyArray [2, 2], NonEmptyArray [1]]
+-- | ```
 group :: forall a. Eq a => NonEmptyArray a -> NonEmptyArray (NonEmptyArray a)
 group = unsafeAdapt $ A.group
 
-group' :: forall a. Ord a => NonEmptyArray a -> NonEmptyArray (NonEmptyArray a)
+-- | Group equal elements of an array into arrays.
+-- |
+-- | ```purescript
+-- | groupAll (NonEmptyArray [1, 1, 2, 2, 1]) ==
+-- |   NonEmptyArray [NonEmptyArray [1, 1, 1], NonEmptyArray [2, 2]]
+-- | `
+groupAll :: forall a. Ord a => NonEmptyArray a -> NonEmptyArray (NonEmptyArray a)
+groupAll = groupAllBy eq
+
+-- | Deprecated previous name of `groupAll`.
+group' :: forall a. Warn (Text "'group\'' is deprecated, use groupAll instead") => Ord a => NonEmptyArray a -> NonEmptyArray (NonEmptyArray a)
```

Comment:
Looks like I forgot to quote groupAll in the deprecation warning of Data.Array.group'. Could you do it there and here?

Reply:
Sure!

Reply:
Done